### DISCUSSION TITLE
Valheim running mods under LGSM FAQ?

### DISCUSSION THREAD

**Discussion by TheWebGamer** [GameServerManagers/LinuxGSM#3336] [Q&A] [answered]
Is there anyone running Mods in Valheim under LGSM?   Is there an FAQ for it someplace I was unable to find?

Many thanks for any assistance.  :)

**Comment by wldwalker** [comment, marked as answer]
At this time it would probably be best to take a look at https://umod.org/community/valheim or jump in the LGSM discord.

**Reply by TheWebGamer** [reply]
Thank you!
